pkg/asset/store.go

```diff
@@ -47,10 +108,19 @@ func (s *StoreImpl) fetch(asset Asset, indent string) error {
 		parents.Add(d)
 	}
 
-	logrus.Debugf("%sGenerating %s...", indent, asset.Name())
-	err := asset.Generate(parents)
+	// Before generating the asset, look if we have it all ready in the state file
+	// if yes, then use it instead
+	logrus.Debugf("%sFetching asset from state file: %s", indent, reflect.TypeOf(asset).String())
```

Comment:
This should either be dropped or be something like `looking up asset from state file`